debunkbot/views.py

```diff
@@ -73,3 +73,40 @@ def claims_tracker(request, claims_db):
             claims_tracker_serializer.save()
             return Response(claims_tracker_serializer.data)
         return Response(claims_tracker_serializer.errors, status=400)
+
+
+@api_view(["GET"])
+@authentication_classes(
+    [
+        TokenAuthentication,
+    ]
+)
+@permission_classes(
+    [
+        IsAuthenticated,
+    ]
+)
+def claims_database_details(request, claims_db_id):
+    gsheet = GSheetClaimsDatabase.objects.filter(id=claims_db_id).first()
+    if gsheet:
+        claim_first_appearance_column_name = gsheet.claim_first_appearance_column_name
+        platform_publication_date_column_name = (
+            gsheet.platform_publication_date_column_name
+        )
+        return Response(
+            {
+                "spreadsheet_id": gsheet.spreadsheet_id,
+                "worksheets": gsheet.worksheets,
+                "headers_row": gsheet.claims_start_row,
+                "claim_appearances_columns": gsheet.claim_url_column_names,
+                "claim_author_column": gsheet.claim_author_column_name,
+                "claim_rating_column_name": gsheet.claim_rating_column_name,
```

Comment:
```suggestion
                "claim_rating_column": gsheet.claim_rating_column_name,
```
Since we're dropping `_name`, lets drop it everywhere?